sdk/src/sdk/vdb/grpc/kuksa_val_v2/BrokerClient.cpp

```diff
@@ -133,147 +176,26 @@ BrokerClient::setDatapoints(const std::vector<std::unique_ptr<DataPointValue>>&
         std::move(batchRequest),
         [result](const kuksa::val::v2::BatchActuateResponse& reply) {
             // Everything went fine, return empty map
+            result->insertResult(SetErrorMap_t());
```

Comment:
everything went fine and we return a SetErrorMap?

Reply:
That's why the map is empty. The original idea was to return errors on signal level: Some signals are set successfully, some don't. The latter ones are put into the error map together with the reason.
The current definition of the broker interface says: Either all signals can be set successfully or none is set.